docs/api/code_check.rst

```diff
@@ -91,8 +91,12 @@ This macro supports the following static analysis tools with their requirements:
 
   * CLANGQUERY_EXECUTABLE is defined and found prior to calling this macro
 
+- clang-tidy
+
+  * CLANGTIDY_EXECUTABLE is defined and found prior to calling this macro
+
 These are added as children to the `check` build target and produce child build targets
-that follow the pattern `<PREFIX>_<cppcheck|clangquery>_check`.
+that follow the pattern `<PREFIX>_<cppcheck|clangquery|clang_tidy>_check`.
```

Comment:
Can you fix the wrong clangquery on this line... it should have been clang_query